samples/nrf9160/gps/src/main.c

```diff
@@ -126,65 +130,92 @@ static int init_app(void)
 					       NRF_GNSS_NMEA_GGA_MASK |
 					       NRF_GNSS_NMEA_RMC_MASK;
 
-	if (setup_modem() != 0) {
-		printk("Failed to initialize modem\n");
-		return -1;
-	}
+	if (ctrl == GNSS_INIT_AND_START) {
+		gnss_fd = nrf_socket(NRF_AF_LOCAL, NRF_SOCK_DGRAM, NRF_PROTO_GNSS);
 
-	gnss_fd = nrf_socket(NRF_AF_LOCAL, NRF_SOCK_DGRAM, NRF_PROTO_GNSS);
+		if (gnss_fd >= 0) {
+			printk("GPS Socket created\n");
+		} else {
+			printk("Could not init socket (err: %d)\n", gnss_fd);
+			return -1;
+		}
 
-	if (gnss_fd >= 0) {
-		printk("Socket created\n");
-	} else {
-		printk("Could not init socket (err: %d)\n", gnss_fd);
-		return -1;
-	}
+		retval = nrf_setsockopt(gnss_fd,
+					NRF_SOL_GNSS,
+					NRF_SO_GNSS_FIX_RETRY,
+					&fix_retry,
+					sizeof(fix_retry));
 
```

Comment:
Nit: Typically an operation and error check is done in a single stream, no need for an empty line in between. IMO that helps to keep the code more concise.

Reply:
That's the original style of the code, just moved them to another function. I'll rather leave it as is if there is not other need for changes.

Reply:
removed the empty lines, because there were other needed changes.